### PR TITLE
Add E2E tests

### PR DESCRIPTION
**Pull request by bahmutov**
Using [cypress.io](https://www.cypress.io/) of course 😌
Blog post about testing this app at https://glebbahmutov.com/blog/testing-svelte-store/

### CODE REVIEW

**Comment by bahmutov** [comment]
One thing - I added Travis CI build but the badge is pointing at my fork, probably should switch to `sveltejs/template-store` CI build.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  directories:
+    - node_modules
+notifications:
+  email: true
+node_js:
+  - '8'
+before_install:
+  - npm install -g yarn@1.3.2
+install:
+  - npm install
```
Comment (bahmutov) [inline .travis.yml:13]: I ultimately gave up - locally `yarn` command did install `run-p` alias from `npm-run-all` but on CI I could not get it to work! `run-p` was always not installed. So I switched to `npm install` with cached `node_modules` for speed.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# svelte app
+# svelte app [![Build Status](https://travis-ci.org/bahmutov/template-store.svg?branch=master)](https://travis-ci.org/bahmutov/template-store)
```
Comment (bahmutov) [inline README.md:1]: this should be switched to this repo after setting up CI build.

**Comment by Rich-Harris** [comment]
Thank you for this, and the blog post. I'm currently trying to get Cypress integrated into the [starter template](https://github.com/sveltejs/sapper-template) for [Sapper](https://sapper-template.now.sh/blog/what-is-sapper). Having some Windows problems, unsurprisingly, but once it's set up there'll be a default way to write acceptance tests for Sapper apps, which is great.

By the way, one wrinkle I ran into: I couldn't get the template to deploy on Now with Cypress as a dev dependency. My workaround is to have it as a global dependency locally, and add a `yarn add cypress` line to my CI scripts. Interested to know if there is a smarter solution than that.

**Comment by bahmutov** [comment]
👍 🎉 100 times @Rich-Harris !

Love it!

- Windows is the black sheep in the family :( Mind looking if your issue is already listed here https://github.com/cypress-io/cypress/issues?q=is%3Aissue+is%3Aopen+label%3A%22OS%3A+windows%22 and if not add new one? This way I can take a look and hopefully fix it
- Hmm, do you install dev dependencies when deploying with `now`? One thing you can do is define an environment variable to skip Cypress binary install (I bet this is what fails) 
```
CYPRESS_SKIP_BINARY_INSTALL=1
```
https://docs.cypress.io/guides/getting-started/installing-cypress.html#Skipping-Installation

Love the `Sapper` idea, and I used Next.js, Nuxt.js before - so your addition is very welcome to me

**Comment by Rich-Harris** [comment]
Thanks!

The Windows issue I'm dealing with isn't a Cypress issue, it's something internal to Sapper that I haven't managed to track down yet.

Yeah, `now` install dev dependencies by default because it executes `npm run build` before `npm start`. Though it [looks like](https://zeit.co/docs/deployment-types/node#ignoring-devdependencies) you can disable that behaviour, which may be what I need. If not, that env variable should do the trick. Thank you!

**Pull request merged**